### PR TITLE
Fix minor mistakes in the book

### PR DESCRIPTION
**Pull request by esclear**
Sorry :/

### CODE REVIEW

File: src/doc/trpl/dining-philosophers.md [hunk 1]
```diff
@@ -540,7 +540,7 @@ fn main() {
         Philosopher::new("Gilles Deleuze", 1, 2),
         Philosopher::new("Karl Marx", 2, 3),
         Philosopher::new("Emma Goldman", 3, 4),
-        Philosopher::new("Michel Foucault", 0, 4),
+        Philosopher::new("Michel Foucault", 4, 0),
```
Comment (alexcrichton) [inline src/doc/trpl/dining-philosophers.md]: How come these changed?

Comment (da-kid) [inline src/doc/trpl/dining-philosophers.md]: They haven't. It is in the explanation.
This is the examples way of trying to evade a deadlock.

Therefore I have reverted it :S

**Comment by steveklabnik** [comment]
Can you squash these two please? r=me after

**Comment by da-kid** [comment]
@steveklabnik Rather all three :)
I am just worried that the pull request gets messed up if I force-push to my repository.

**Comment by steveklabnik** [comment]
it won't! github updates it just fine.

**Comment by da-kid** [comment]
\o/

**Comment by da-kid** [comment]
r=steveklabnik

**Comment by da-kid** [comment]
\o/
Well, I'm a bit ashamed right now.
r= @steveklabnik

**Comment by da-kid** [comment]
Okay, I don't get it. Shouldn't @bors do something now? Help!

**Comment by steveklabnik** [comment]
@da-kid bors only listens to people with commit bit, sorry about that! you did it correctly otherwise :)

**Pull request merged**